tests/unit/cn/cn_ingest_test.c

```diff
@@ -278,9 +278,11 @@ MTF_DEFINE_UTEST_PRE(cn_ingest_test, worker, test_pre)
     err = cn_tree_create(&cn.cn_tree, NULL, 0, &cp, &mock_health, &rp);
     ASSERT_EQ(err, 0);
 
+    /* The kblocks contain no keys, so the ingest should fail.
+     */
     init_mblks(m, n_kvsets, &k, &v);
     err = cn_ingestv(cnv, mbv, NELEM(cnv), U64_MAX, U64_MAX, NULL, NULL);
-    ASSERT_EQ(EINVAL, merr_errno(err));
+    ASSERT_NE(0, err);
```

Comment:
I kind of like the pattern of `ASSERT_NE(0, merr_errno(err))`. It is a much more useful error message when you see the errno value rather than a 64 bit integer.

Reply:
Well, if I have the value of err I can decode it manually to see where the error was generated.  But we should actually have assert macros that take a merr_t and decode it for us, now that would be very very useful...